### ISSUE TITLE
OPDS: Make it easier to just read a book

### ISSUE THREAD

**Issue by bertob** [johnfactotum/foliate#405]
Currently in order to read a book found in a Catalog, you have to

- Click the cover
- Click the "free" button
- Choose a format
- Browser opens
- Open the file with Foliate
- The book opens in Foliate

![image](https://user-images.githubusercontent.com/1908896/83307609-68805b00-a205-11ea-910c-79c4fe8b565d.png)

I think this could be streamlined to something like this:

- Click the cover
- Click the "Read" button
- The book opens in Foliate

Basically, Foliate would just pick the best format by default, and automatically download books (either into its Flatpak directory, or something user-visible e.g. `~/Documents/Books`).

In order to make other formats available there could be a secondary button that just links to the website, or a format dropdown popover next to the "Read" button (so you can choose a format if you want, but get the recommended default if you don't do anything).

**Comment by johnfactotum** [comment]
It should probably also remember that the book has been downloaded, and when you navigate to the same book again, you should be able to open the local file instead of downloading it again.
